storage/statedb/cache_redis.go

```diff
@@ -97,3 +99,15 @@ func (cache *RedisCache) Has(k []byte) ([]byte, bool) {
 	}
 	return val, true
 }
+
+func (cache *RedisCache) Publish(channel string, msg string) error {
+	return cache.client.Publish(channel, msg).Err()
+}
+
+func (cache *RedisCache) Subscribe(channel string) *redis.PubSub {
+	return cache.client.Subscribe(channel)
```

Comment:
By checking the AWS document, I've noticed that this might not respond immediately and should use `Receive()` function to do so. How was it when you tested? Are we planning to have a routine for receiving the response.

Reply:
We can use a `Receive()` method or channel to receive messages. `Recieve()` method returns only one subscribing message and the type of message could be unnecessary data. So, I added one more method `(cache *RedisCache) SubscriptionChannel` and the usage is placed in `_TestSubscription`. 
You may use the channel like the following.
```
ch := cache.SubscriptionChannel(channelName)
for msg := range ch {
    // handle msg.Payload
}
```